admin_manual/installation/deployment_considerations.rst

```diff
@@ -43,6 +43,8 @@ Provider setup:
   requests accordingly (2-n)
 * GPFS or Ceph via phprados (2-n, 3 to be safe, Ceph 10+ nodes to see speed 
   benefits under load)
+* If you are scaling, you have to be sure that the config.php in all the severs are the same. 
+  Otherwise you will run into problems.
```

Comment:
I am not quite sure about this sentcence. Scaling and clustering are different things.
1. There are applications environments where applications can run on multiple servers in parallel doing the same task (here you can add more servers to scale, if one breaks, the app continues)
2. There are application environments where you have multiple servers but the app can only run different tasks but not the same one (here you can add more servers to scale, if one breaks, the app dies and restarts with a different task)
3. There are application environments where you only can run an app on an active server and spare servers are waiting for a situation to takeover (here you only can exchange the server by a new one to scale for more performance)

Now my believe is, that php is by design #.2 (partially) and #.3 (fully) which is clustering

Your sentence should be changed to reflect this like:
``In case of clustering, your cluster nodes must have the same owncloud configuration including an identical config.php to avoid any potential issues.``

Reply:
That sounds about right to me.